### ISSUE TITLE
Compatibility with Chrome 65?

### ISSUE THREAD

**Issue by orschiro** [eliskah/tab-numbers#2]
Hey @eliskah 

Just came across your extension and really like the idea!

Loaded the unpackaged extension but don't see any number when pressing Alt or Ctrl. 

Can you confirm it's still compatible?

Thank you!

**Comment by eliskah** [comment]
Hi @orschiro! I am so glad you find the extension useful!

I've just tested on Chrome `65.0.3325.162` on Mac OSX `10.13.3` and for me it works as expected. This is what I did to test:

* Downloaded the extension `zip` file using Github's `Clone or download` and unarchived
* In browser, navigated to `chrome://extensions/`, with Developer Mode switched on, clicked `Load Unpacked` and loaded the unpacked folder
* After that, I skipped to some other open tab, pressed `ALT+T` and that's the result:

![screen shot 2018-03-18 at 23 04 57](https://user-images.githubusercontent.com/1964407/37571737-d2ade11a-2b00-11e8-86f0-9501ac614747.png)

Could you please try to retry these steps, and if it still does not work for you, could you tell me more about your configuration?

**Comment by orschiro** [comment]
Hey @eliskah 

> After that, I skipped to some other open tab, pressed ALT+T and that's the result:

My fault! You're correct. It's working that way. I expected to see tab numbers by just pressing Alt instead of Alt + T. Is it possible to change the shortcut to just Alt?

**Comment by eliskah** [comment]
@orschiro In this extension, [chrome.commands](https://developer.chrome.com/apps/commands) are used to trigger displaying the index and additional letter or number is required to be used. Users can even change the commands themselves in `chrome://extensions/configureCommands`but if you try this, you'll see that `Alt` itself is not a valid shortcut :(

I think I remember that using only `Alt` or better `CMD` was my original idea back when creating the extension, but I could not make it work for this exact reason.

**Comment by orschiro** [comment]
> I think I remember that using only Alt or better CMD was my original idea back when creating the extension, but I could not make it work for this exact reason.

Thanks! Do you have any idea how the following extension does it?

https://chrome.google.com/webstore/detail/tab-number/fijaenjgknobfdombbdchngpamggajpm

**Comment by eliskah** [comment]
Hi @orschiro ... Thanks for showing me the other extension, it's funny I did not found it before doing this one ...

I've checked the source code by opening the background page (available from `chrome://extensions`) and by checking the content script in inspector (in Chrome inspector, I clicked `Sources` and `Content Scripts`).

It looks to me that the other extension is using a content script (`num_tabs.js`) that is injected to any page. This content script is **listening to any key pressed/released** event with `document.onkeydow` and `document.onkeyup`. If the pressed key is of interest (`CMD` or `CTRL`/`ALT`) so the browser should change favicons, the content script sends message to the background script (`background.js`) of the extension. The message is `ctrl_hold` or `ctrl_release`. Based on that, the background page switches all the favicons in any tab.

My extension is **listening to the keyboard shortcut of defined commands (`ALT+T` on default)**. When this event happens, it triggers the change of all titles by prepending the index to existing title.
